app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -2128,9 +2138,6 @@ protected function _saveProductWebsites(array $websiteData)
      */
     protected function _saveStockItem()
     {
-        /** @var $stockResource \Magento\CatalogInventory\Model\ResourceModel\Stock\Item */
-        $stockResource = $this->_stockResItemFac->create();
```

Comment:
Looks like it was the only usage of `_stockResItemFac` property in the class.
In this case it may be deprecated.

Reply:
Done.